src/components/Calculator.css

```diff
@@ -46,3 +46,7 @@ button {
 button:hover {
   background-color: #ddd;
 }
+
+.button.zero-button {
+width: 9rem;
+}
```

Comment:
- [ ] Excellent job implementing the previous reviewer's suggestions 👏. However, after making the changes, the equal button has shifted slightly to the right. So make sure it's properly aligned with those on top, and I'd highly recommend updating the **width** to **8.75rem** to give your UI a nicer look.  :ok_hand:  😊 

```
.button.zero-button {
   width: 8.75rem;
}
```

#### Your calculator currently :point_down: 

![Screenshot from 2023-08-07 15-57-00](https://github.com/AlyAtalla/math-magician/assets/89228743/0b0f8f99-ca0d-49bb-b104-14476c184d40)


#### Your calculator when you update `button.zero-button`  `8.75rem` :point_down: 
![Screenshot from 2023-08-07 16-28-01](https://github.com/AlyAtalla/math-magician/assets/89228743/72b8c32f-db46-468c-8b97-b96905e5366e)

Reply:
thank you